curvefs/proto/metaserver.proto

```diff
@@ -313,11 +313,13 @@ message AppendS3ChunkInfoRequest {
     required uint64 inodeId = 5;
     map<uint64, S3ChunkInfoList> s3ChunkInfoAdd = 6;
     map<uint64, S3ChunkInfoList> s3ChunkInfoRemove = 7;
+    required bool returnInode = 8;
```

Comment:
GetOrModifyChunkInfoReuqest?

AppendS3ChunkInfoRequest  is confused. it is responsible for a lot of functionality, not just Append.

Reply:
done